### DISCUSSION TITLE
Get starstruck

### DISCUSSION THREAD

**Discussion by Ysrbolles** [Schweinepriester/github-profile-achievements#194] [General]
Hi all,
for get starstruck, need to reach 16 star. If u want u can put star on my readme repository, I will do the same for you.

Add a answer with your profile repository for have a chance to get the star from other user.

https://github.com/Ysrbolles/Ysrbolles

**Comment by Luscarvalho** [comment]
https://github.com/Luscarvalho/Luscarvalho

**Reply by Ysrbolles** [reply]
Already done

**Reply by nel0x** [reply]
@Luscarvalho done. Can you please star my project https://github.com/nel0x/arch-installer too

**Reply by Ysrbolles** [reply]
@nel0x done

**Reply by Crushoverride007** [reply]
join the troops mates. already starred yours, please leave a star here https://github.com/Crushoverride007/Crushoverride007

**Reply by melvincwng** [reply]
@Ysrbolles , would appreciate if you could help ⭐: https://github.com/melvincwng/facebook-clone

Thanks!

**Reply by guexs** [reply]
@Ysrbolles done

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by OmarHamdann** [reply]
done, pls star here https://github.com/OmarHamdann/MERAKI_Academy_Project_5

**Reply by ArielMAJ** [reply]
@Luscarvalho I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@nel0x I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by melvincwng** [comment]
For those who are interested to help ⭐:

1. https://github.com/melvincwng/facebook-clone
2. https://github.com/Your-First-Open-Source-Project/Your-First-Open-Source-Project

Thanks In advance, glad to see such a vibrant and inclusive community here on GitHub 😊

**Reply by Crushoverride007** [reply]
done, please leave a star here https://github.com/Crushoverride007/Crushoverride007

**Reply by OmarHamdann** [reply]
done, pls star here https://github.com/OmarHamdann/MERAKI_Academy_Project_5

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by WHAHA-HA** [reply]
@melvincwng 
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
@melvincwng, @Crushoverride007 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by OmarHamdann** [comment]
https://github.com/OmarHamdann/MERAKI_Academy_Project_5

thanks 😊

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by nel0x** [reply]
@OmarHamdann Starred :D
Please also leave one here https://github.com/nel0x/arch-installer

**Reply by WHAHA-HA** [reply]
@OmarHamdann 
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
@OmarHamdann I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by guexs** [comment]
Thanks! 
https://github.com/wannderlust/wannderlust

**Reply by OmarHamdann** [reply]
done, pls star here https://github.com/OmarHamdann/MERAKI_Academy_Project_5

**Reply by guexs** [reply]
Done

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by WHAHA-HA** [reply]
@zuckgs 
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
@guexs I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by number0x0001** [comment]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by noowz** [comment]


**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by WHAHA-HA** [reply]
@Fastxyz 
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
@Fastxyz I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by erikn69** [comment]
Done,
please help here: https://github.com/erikn69/laravel-google-drive-ext-demo

**Reply by kshitij-halankar** [reply]
@erikn69 Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by ArielMAJ** [reply]
@erikn69 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by danielr1996** [comment]
https://github.com/danielr1996/kubectl-action

**Comment by kshitij-halankar** [comment]
Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by WHAHA-HA** [reply]
@kshitij-halankar 
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Comment by WHAHA-HA** [comment]
@Ysrbolles 
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Comment by number0x0001** [comment]
Please leave a star, thanks: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by ArielMAJ** [comment]
@Ysrbolles I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.
